packages/webapp/public/locales/en/common.json

```diff
@@ -59,7 +59,7 @@
   "SAVE_CHANGES": "Save Changes",
   "SEARCH": "Search",
   "SELECT": "Select",
-  "SELECT_ALL_COUNT": "Select all {{count}}",
+  "SELECT_ALL": "Select all",
```

Comment:
I am kind of surprised by Loic's decision here. I at first thought the count was maybe useful for differentiating between "selecting all visible" and "selecting all". 

But now that I test it both the table header checkbox and the action bar select all both are "selecting all visible". So now it makes me think action bar select all is not necessary and a duplicate functionality haha.

Reply:
> the count was maybe useful for differentiating between "selecting all visible" and "selecting all".

True...

I appreciate the action bar select all on mobile view, but I agree that it's not necessary on desktop!